### DISCUSSION TITLE
Go to location

### DISCUSSION THREAD

**Discussion by chrismelba** [Tasshack/dreame-vacuum#237] [Ideas]
With my xiaomi robots I am able to send them to the bin for me to empty after a clean. Is it possible to add this for the dreame?

I have thought that I could do a spot clean where I want then stop, but my robot will always wet the mops prior to any clean, so then I must dry them again.

**Comment by chrismelba** [comment]
Looks like this is already in the beta

**Reply by Tasshack** [reply]
If you have a vacuum with camera, integration uses the internal `Cruising to a point` feature as `goto` service but if your device does not have a camera vacuum cannot be sent to a point from API. Because of that integration uses the `Zone Cleaning` feature and sends `Stop` command when vacuum is reached to the specified zone on devices does not have the `Cruising` feature, if that was your case this won't solve your wet mop problem.
Dreame does not strictly have a `go to` feature unlike Roborock or some other brands,

**Reply by Tasshack** [reply]
I have also opened a pull request for this feature to be working with the map card. Currently you cannot use this service with the map card util this pull request is closed.
https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/pull/650

**Reply by chrismelba** [reply]
But with the next beta (or the one just release?) my vacuum should be able to be used in sweep only mode?
